### PR TITLE
Update useEffect dependency

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description
Update dependency

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ofpiyush — APPROVED** [review]

**Comment by hetunandu** [comment]
Please add some more info about why this change was needed

**Comment by ofpiyush** [comment]
@hetunandu We were tracking props that didn't need to be tracked. It's a minor thing I noticed when fixing a merge conflict.

File: app/client/src/components/editorComponents/EditableText.tsx [hunk 1]
```diff
@@ -106,11 +106,11 @@ export const EditableText = (props: EditableTextProps) => {
 
   useEffect(() => {
     setIsEditing(!!props.isEditingDefault);
-  }, [props.defaultValue, props.isEditingDefault, setValue]);
```
Comment (akash-codemonk) [inline app/client/src/components/editorComponents/EditableText.tsx:109]: @ofpiyush There seems to be an issue, when the editabletext shows an error and the user switches to another action. The editabletext continues to have a border. I am guessing `props.defaultValue` should stay here as a dependency

Comment (ofpiyush) [reply app/client/src/components/editorComponents/EditableText.tsx:109]: Oh wow you're correct!

**Review by ofpiyush — APPROVED** [review]

**Pull request merged**